lua/mason-nvim-dap/mappings/adapters.lua

```diff
@@ -0,0 +1,57 @@
+local settings = require('mason-nvim-dap.settings')
+
+local M = {}
+
+-- https://github.com/mfussenegger/nvim-dap/wiki/Debug-Adapter-installation#go-using-delve-directly
+M.delve = {
+	type = 'server',
+	port = '${port}',
+	executable = {
+		command = 'dlv',
+		args = { 'dap', '-l', '127.0.0.1:${port}' },
+	},
+}
+
+M.python = {
+	type = 'executable',
+	command = vim.fn.exepath('python3'),
+	args = { '-m', 'debugpy.adapter' },
+}
+
+M.node2 = {
+	type = 'executable',
+	command = 'node',
+	args = { vim.fn.stdpath('data') .. '/mason/packages/node-debug2-adapter/out/src/nodeDebug.js' },
+}
+
+M.chrome = {
+	type = 'executable',
+	command = 'node',
+	args = {
+		vim.fn.stdpath('data') .. '/mason/packages/chrome-debug-adapter/out/src/chromeDebug.js',
+	},
```

Comment:
This should be `command = 'chrome-debug-adapter'` with no args (the command runs the correct javascript file)